File: BE-FindingJob/src/main/java/com/example/befindingjob/dto/auth/LoginResponse.java
package com.example.befindingjob.dto.auth;

import com.example.befindingjob.entity.enumm.Role;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class LoginResponse {
    private String token;
    private int userId;
    private Role role;
    private String fullName;
}

File: JobHub/app/src/main/java/com/example/jobhub/dto/auth/LoginResponse.kt
package com.example.jobhub.dto.auth

import android.os.Parcelable
import com.example.jobhub.entity.enumm.Role
import kotlinx.parcelize.Parcelize

@Parcelize
data class LoginResponse(
    val token: String,
    val userId: Int,
    val role: Role,
    val fullName: String
) : Parcelable
